engine/proofview.ml

```diff
@@ -905,11 +905,11 @@ let tclPROGRESS t =
   if not test then
     tclUNIT res
   else
-    tclZERO (CErrors.UserError (Some "Proofview.tclPROGRESS" , Pp.str"Failed to progress."))
+    tclZERO (CErrors.UserError (Some "Proofview.tclPROGRESS", Pp.str "Failed to progress."))
 
-exception Timeout
+exception PV_Timeout
```

Comment:
Instead of creating a new exception, I think you should rather share the one from the logic monad. It doesn't really make sense to define a fresh exception whose sole purpose is to encapsulate the other one...

Reply:
I am not creating a new exception, just renaming it, right?

Reply:
Sure I could try to tweak this but I'm not sure how much I understand the logic going on here, maybe that should go on a separate PR?

Reply:
I'm just asking for you to write
```
exception PV_Timeout = LM_Timeout
```

Reply:
What should we do with the error messages?

Reply:
Ok I've done what I could.